SU-SWS/stanford_story_page
# HSDO-1198 Changed default paragraph image

# READY FOR REVIEW

# Summary
- Changed the image for the default paragraph layout
- rebuilt the feature so that it get the correct stuff with new fields on the icon paragraph type.

# Needed By (Date)
- Some Day

# Urgency
- Low

# Steps to Test

1. Checkout branch
2. Install/revert stanford_story_page
3. create a new story page and click to use the "Home Page" default
4. ensure the image of the boy sitting on the edge of a building is no longer there. (looks like hes about to jump)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
